cluster/gce/gci/configure-helper-common.sh

```diff
@@ -1424,7 +1424,112 @@ scrape_configs:
     static_configs:
     - targets: ['127.0.0.1:2379','127.0.0.1:8080','127.0.0.1:10251','127.0.0.1:10252']
 EOF
-  ./prometheus --config.file="/tmp/prometheus-metrics.yaml" --web.listen-address=":9090" --web.enable-admin-api &
+  nohup ./prometheus --config.file="/tmp/prometheus-metrics.yaml" --web.listen-address=":9090" --web.enable-admin-api > prometheus.log 2>&1 &
+
+}
+
+
+function collecting-pprof {
+  local ports=$1
+  local name=$2
+  echo "curl http://127.0.0.1:${ports}/debug/pprof/profile -o ${name}-profile-${CURRENTTIME}.pprof"
+  curl http://127.0.0.1:${ports}/debug/pprof/profile -o ${name}-profile-${CURRENTTIME}.pprof
+  echo "curl http://127.0.0.1:${ports}/debug/pprof/heap -o ${name}-heap-${CURRENTTIME}.pprof"
+  curl http://127.0.0.1:${ports}/debug/pprof/heap -o ${name}-heap-${CURRENTTIME}.pprof
+  echo "curl http://127.0.0.1:${ports}/debug/pprof/goroutine -o ${name}-goroutine-${CURRENTTIME}.pprof"
+  curl http://127.0.0.1:${ports}/debug/pprof/goroutine -o ${name}-goroutine-${CURRENTTIME}.pprof
+  echo "curl http://127.0.0.1:${ports}/debug/pprof/mutex -o ${name}-mutex-${CURRENTTIME}.pprof"
+  curl http://127.0.0.1:${ports}/debug/pprof/mutex -o ${name}-mutex-${CURRENTTIME}.pprof
+  echo "curl http://127.0.0.1:${ports}/debug/pprof/block -o ${name}-block-${CURRENTTIME}.pprof"
+  curl http://127.0.0.1:${ports}/debug/pprof/block -o ${name}-block-${CURRENTTIME}.pprof
+  echo "curl http://127.0.0.1:${ports}/debug/pprof/threadcreate -o ${name}-threadcreate-${CURRENTTIME}.pprof"
```

Comment:
ideally, all these http://127.0.0.1 should be placed into a variable to prevent repeat

Reply:
okay to leave it if in a hurry. please come back and refactor when there's time.

this comment resolved.

Reply:
good suggestion. I have replaced with variable.

Reply:
thanks. please resolve this.